### ISSUE TITLE
RTT plugin won't connect to J-Link on OS X

### ISSUE THREAD

**Issue by skot** [vovkos/ioninja-issues#10]
With new ioninja versions 3.12.2 and 3.14.1 when I try and connect to my Segger J-Link using the RTT plugin I get the following error;

```
Cannot establish J-Link RTT connection: dlopen(/Applications/SEGGER/JLink/libjlinkarm.dylib, 5): no suitable image found.  Did find:
 /Applications/SEGGER/JLink/libjlinkarm.dylib: code signature in (/Applications/SEGGER/JLink/libjlinkarm.dylib) not valid for use in process using Library Validation: mapping process and mapped file (non-platform) have different Team IDs
 /Applications/SEGGER/JLink/libjlinkarm.dylib: stat() failed with errno=22
```

This does not occur with ioninja 3.12.0
macOS 10.14.6, MacBook Pro

**Comment by vovkos** [comment]
Thanks for the report. Good catch. 

Starting with IO Ninja ``3.14.0`` we sign and notarize app bundles with Apple as to avoid macOS Gatekeeper warnings. In order to do that, it is required by Apple to turn on so-called Runtime Hardening, which among other things includes Library Validation. This Library Validation is what prevented IO Ninja to load the JLink dylib.

We have just released a patch ``3.14.1-a`` which disables this Library Validation, but keeps the rest of Runtime Hardening facilities enabled. It appears that this is enough to pass Apple notarization, so that's how we will build our binaries from now on.

Please install this patch (``3.14.1-a``) and try again. You can either download it from https://ioninja.com/downloads.html or update it from Homebrew (version didn't increase, so use ``reinstall``, not ``upgrade``).

Let me know if it works for you.

**Comment by skot** [comment]
nice, `3.14.1-a` works great! Thanks for the fast response.
